logstash-core/lib/logstash/java_pipeline.rb

```diff
@@ -225,11 +225,14 @@ def start_workers
       config_metric.gauge(:graph, ::LogStash::Config::LIRSerializer.serialize(lir))
       config_metric.gauge(:cluster_uuids, resolve_cluster_uuids)
 
-      @logger.info("Starting pipeline", default_logging_keys(
+      pipeline_log_params = default_logging_keys(
         "pipeline.workers" => pipeline_workers,
         "pipeline.batch.size" => batch_size,
         "pipeline.batch.delay" => batch_delay,
-        "pipeline.max_inflight" => max_inflight))
+        "pipeline.max_inflight" => max_inflight)
+      pipeline_log_params["pipeline.sources"] = pipeline_source_details
```

Comment:
Can you also add this to https://github.com/elastic/logstash/blob/55783109493adf1bdc425ad4695207fce60bc447/logstash-core/lib/logstash/java_pipeline.rb#L112